src/components/ToDo/styles/LastRow.styled.ts

```diff
@@ -0,0 +1,70 @@
+import styled, { css } from 'styled-components'
+
+export const Rectangle = styled.div<{ isDarkMode: boolean }>`
+  height: 30px;
+  background-color: ${(props) => (props.isDarkMode ? 'hsl(235, 24%, 19%)' : 'hsl(0, 0%, 98%)')};
+  border-radius: 0px;
+  color: ${(props) => (props.isDarkMode ? 'hsl(234, 39%, 85%)' : 'hsl(235, 19%, 35%)')};
+  box-shadow: ${(props) => props.isDarkMode && '2px 2px 4px #000000'};
```

Comment:
bring box-shadow in, to right before `2px 2px..` so this line will only appear when isDarkMode is true
ie `${(props) => props.isDarkMode && 'box-shadow: 2px 2px 4px #000000;'}`